README.md

```diff
@@ -1 +1,23 @@
-# dissolution-api
\ No newline at end of file
+# Dissolution API
+
+## Technologies
+- [OpenJDK 11](https://jdk.java.net/archive/)
+- [Maven](https://maven.apache.org/download.cgi)
+- [Spring Boot](https://spring.io/projects/spring-boot)
+- [Swagger OpenAPI](https://swagger.io/docs/specification/about/)
+
+### Running locally with Docker  
+
+Required tools:
+- [Docker for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
+- [Docker-Compose](https://docs.docker.com/compose/install/)
+
+To bring the environment up, run `docker-compose up` in the project folder. You must be connected to CH VPN in order to download dependencies.
```

Comment:
Just make sure to include the AWS login step, otherwise docker won't be able to pull the images from ECR